src/alumni/bio.js

```diff
@@ -0,0 +1,86 @@
+import React, { useState, useEffect } from 'react';
```

Comment:
instead of bio.js let's call this alumnit-detail.component.js, and then call your Bio function AlumniDetail